src/main/java/com/mongodb/hibernate/jdbc/MongoConnectionProvider.java

```diff
@@ -34,52 +36,40 @@
 import org.hibernate.HibernateException;
 import org.hibernate.cfg.JdbcSettings;
 import org.hibernate.engine.jdbc.connections.spi.ConnectionProvider;
+import org.hibernate.service.UnknownServiceException;
 import org.hibernate.service.UnknownUnwrapTypeException;
 import org.hibernate.service.spi.Configurable;
+import org.hibernate.service.spi.ServiceRegistryAwareService;
+import org.hibernate.service.spi.ServiceRegistryImplementor;
 import org.hibernate.service.spi.Stoppable;
 import org.jspecify.annotations.Nullable;
 
 /**
  * {@linkplain com.mongodb.hibernate.dialect.MongoDialect MongoDB dialect}'s customized JDBC {@link ConnectionProvider}
  * SPI implementation.
  *
- * <p>{@link MongoConnectionProvider} uses the following Hibernate properties:
- *
- * <table>
- *     <tr><th>Property</th><th>Description</th><th>Required</th></tr>
- *     <tr>
- *         <td>{@value JdbcSettings#JAKARTA_JDBC_URL}</td>
- *         <td>MongoDB
- *         <a href="https://www.mongodb.com/docs/manual/reference/connection-string/">connection string</a>,
- *         which must specify the database name for authentication
- *         if {@value JdbcSettings#JAKARTA_JDBC_USER} is specified.</td>
- *         <td>✓</td>
- *     </tr>
- *     <tr>
- *         <td>{@value JdbcSettings#JAKARTA_JDBC_USER}</td>
- *         <td>{@code userName} for {@link com.mongodb.MongoCredential#createCredential(String, String, char[])}</td>
- *         <td></td>
- *     </tr>
- *     <tr>
- *         <td>{@value JdbcSettings#JAKARTA_JDBC_PASSWORD}</td>
- *         <td>{@code password} for {@link com.mongodb.MongoCredential#createCredential(String, String, char[])}</td>
- *         <td></td>
- *     </tr>
- * </table>
+ * <p>{@link MongoConnectionProvider} uses the {@value JdbcSettings#JAKARTA_JDBC_URL} to create a MongoDB <a
```

Comment:
[optional]

I believe, it should be either

```java
uses the {@value JdbcSettings#JAKARTA_JDBC_URL} configuration property to create
```
or
```java
uses {@value JdbcSettings#JAKARTA_JDBC_URL} to create
```
but not `the {@value JdbcSettings#JAKARTA_JDBC_URL}`.

Reply:
of course you are right. I chose the first option.